src/index.tsx

```diff
@@ -8,7 +8,7 @@ import { App } from './App';
 
 createRoot(document.getElementById('root') as HTMLDivElement)
   .render(
-    <Router>
+    <HashRouter>
```

Comment:
HashRouter as Router
Your HashRouter was called like Router. Here you just changed the name to a longer one